src/plugins/auto/plugin.cpp

```diff
@@ -956,7 +956,7 @@ std::vector<DeviceInformation> MultiDeviceInferencePlugin::FilterDeviceByNetwork
     });
 
     // If CPU is in candidate list, load dynamic network to CPU first
-    if ((model->is_dynamic() || isStateful()) && cpuiter != metaDevices.end()) {
+    if ((model->is_dynamic() || (isStateful() && !isMulti)) && cpuiter != metaDevices.end()) {
```

Comment:
Please check if AUTO folder has only one MULTI flag. If no, please update to single one.

Reply:
Create JIRA: https://jira.devtools.intel.com/browse/CVS-103669 to follow up.